### ISSUE TITLE
Prevent you from punching your friends

### ISSUE THREAD

**Issue by d1gress** [S-B99/KAMI#163]
**Is your feature request related to a problem? Please describe.**
I hate it when I have friends then I hit them because I don't have friends and they're just ghosts haunting me.

**Describe the solution you'd like**
Stop me from hitting ghosts

**Describe alternatives you've considered**
Spiritual cleansing

**Additional context**
cancel packets if you are gonna hit a friend and also exempt them from combat modules

**Comment by S-B99** [comment]
Could you rephrase this please? I believe modules already check if someone is a friend

**Comment by Vonr** [comment]
oh there is one? well then i just want something that prevents me from clicking on a friend and dealing damage

**Comment by S-B99** [comment]
Uh, that would require like a mixin and lots of work, aura for sure ignores friends. I'll leave this open but it's low priority honestly.

**Comment by cookiedragon234** [comment]
Can be done in packets, doesnt require mixins

**Comment by S-B99** [comment]
Added antifriend hit
